app/views/api/v3/analytics/user_analytics/show.html.erb

```diff
@@ -89,66 +90,57 @@
             %>
           </div>
         <% end %>
-
         <%= render(Reports::ProgressControlComponent.new(@service)) %>
-
         <div class="card">
           <a href="#" onclick="openWindow('progress-cohort', 'progress-start'); return false" class="nav-next">
             Cohort report
           </a>
         </div>
-
-        <% if @achievements.show? %>
```

Comment:
Do we want to preserve an equivalent check to this one?

Reply:
@vkrmis that's a good question. I believe we shouldn't. The achievements card should always be displayed even if the facility is brand new. The old `.show?` check tested to see if the facility had `<10` follow-ups to see if the section was displayed.